### ISSUE TITLE
No warning that variable for col parameter in mapping has to be nonnumeric.

### ISSUE THREAD

**Issue by jschepers** [unfoldtoolbox/UnfoldMakie.jl#56]
When running this code, `col` is just ignored because the values in the channel column are numeric. It would be nice if there was a warning/error in this case that says that the values have to be non-numeric.
```julia
plot_erp(effs;  mapping=(; y=:yhat,group= :channel,col=:channel))
```

**Comment by vladdez** [comment]
@behinger Is it fine if we just automatically convert values of :col to non-numeric?

**Comment by behinger** [comment]
should be, only issue is if one puts a numeric variable with many values by accident and then receives 1000 columns :D

**Comment by vladdez** [comment]
fixed #222
